packages/react-chat/src/styles/theme.ts

```diff
@@ -23,69 +21,72 @@ export interface FontOptions {
   height?: BaseCSS['lineHeight'] | Token<typeof Font['LINE_HEIGHTS']>;
 }
 
-export const { styled, config, keyframes, theme, createTheme } = createStitches({
-  ...(__USE_SHADOW_ROOT__ && { root: shadowRoot }),
-
-  theme: {
-    colors: Color.PALETTE,
-    shadows: Color.SHADOWS,
-
-    space: {
-      1: '4px',
-      2: '6px',
-      3: '12px',
-      4: '16px',
-      5: '20px',
-      6: '24px',
-    },
-    sizes: {
-      xxs: '16px',
-      xs: '24px',
-      sm: '36px',
-      md: '42px',
-      lg: '56px',
-      xl: '60px',
-      xxl: '72px',
+export const getDefaultTheme = () => {
+  return {
```

Comment:
could we just do `() => ({})` instead of `() => { return {} }`

Reply:
for sure, I think I was testing with some code inside this closure